src/hooks/sdk/docs/useBandwidthMetrics.stories.mdx

```diff
@@ -6,6 +6,8 @@ The `useBandwidthMetrics` hook returns the available outgoing and incoming bandw
 
 This hook re-renders frequently. You should avoid using it high in your app tree, and ideally use it in your leaf components.
 
+**Please note** - This hook will eventually be deprecated, as we have a new hook `useMediaStreamMetrics` to return media stream metrics which include the available outgoing and incoming bandwidth.
```

Comment:
Should be this hook is deprecated and will be removed in future version, please use `useMediaStreamMetrics`

Reply:
Thats a good call, we should add a log warning in useBandwidthMetric hook.

Reply:
you mean create a console warning in `useBandwidthMetric` hook?

Reply:
Yes.